modules/openapi-generator/src/main/resources/kotlin-client/IStorable.mustache

```diff
@@ -0,0 +1,9 @@
+{{>licenseInfo}}
```

Comment:
@4brunu Good spot, it's definitely out of place. We had an issue with where to locate this file as it was a "supporting file" but we were trying to understand the system. I'll relook at it's location. 

As for the size of the PR, apologies if it's unwieldy - i wasn't sure how to approach this. We landed on trying to provide a largely functional library which made it quite large. We have tried to extend but not modify the system as much as possible in both code and templates to minimise the size and maintainability of the library. I'm guessing most PR's are more minor additions and full additional libraries or generators or less common. Maybe we can add to the contributors guidelines with some information for those adding a whole new library. e.g as i type this i wonder if a convenient way for you to build this would have been for me to provide you with a dockerfile that could set up the build environment. I'm having a look for any appropriate docker images now